tests/test_outfit_endpoint.py

```
@@ -0,0 +1,66 @@
+        """When no request style and no profile, falls back to Casual."""
+        mock_client = MagicMock()
+        mock_client.table.return_value.select.return_value.eq.return_value.single.return_value.execute.return_value.data = None
+        mock_get_client.return_value = mock_client
+        result = _resolve_style("user-123", None)
+        assert result == "Casual"
+
+
+class TestOutfitEndpoint:
+    @patch("server.services.rules_engine.get_eligible_items", return_value=[])
+    @patch("server.routers.outfit.get_supabase_client")
+    def test_suggest_returns_correct_shape(self, mock_get_client, mock_get_items):
```

Ruff (0.15.5)

[warning] Unused method argument: `mock_get_items`

(ARG002)